cmake/information_helpers.cmake

```diff
@@ -18,12 +18,12 @@ macro(ginkgo_git_information)
                 OUTPUT_VARIABLE GINKGO_GIT_BRANCH
                 OUTPUT_STRIP_TRAILING_WHITESPACE)
             execute_process(
-                COMMAND ${GIT_EXECUTABLE} log -1 --format=%H ${Gingko_SOURCE_DIR}
+                COMMAND ${GIT_EXECUTABLE} log -1 --format=%H ${Ginkgo_SOURCE_DIR}
```

Comment:
So we didn't catch this before since it is already being executed in Ginkgo_SOURCE_DIR?

Reply:
Yes, I guess so.